### PR TITLE
Ported UI docker container tests

### PR DESCRIPTION
**Pull request by abalakh**
Depends on SatelliteQE/airgun#296
Test results:
```python
pytest -v tests/foreman/ui_airgun/test_container.py
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-4.0.2, py-1.6.0, pluggy-0.8.1 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: xdist-1.26.1, services-1.3.1, mock-1.10.0, forked-0.2, env-0.6.2, cov-2.6.1
collecting ... 2019-02-27 18:28:07 - conftest - DEBUG - BZ deselect is disabled in settings

collected 6 items

tests/foreman/ui_airgun/test_container.py::test_positive_create_with_external_compresource PASSED [ 16%]
tests/foreman/ui_airgun/test_container.py::test_positive_create_with_compresource PASSED [ 33%]
tests/foreman/ui_airgun/test_container.py::test_positive_power_on_off PASSED [ 50%]
tests/foreman/ui_airgun/test_container.py::test_positive_create_with_external_registry PASSED [ 66%]
tests/foreman/ui_airgun/test_container.py::test_positive_delete PASSED   [ 83%]
tests/foreman/ui_airgun/test_container.py::test_positive_create_with_unix_socket PASSED [100%]

=================== 6 passed, 8 warnings in 1084.28 seconds ====================
```

### CODE REVIEW

File: tests/foreman/ui_airgun/test_container.py [hunk 1]
```diff
@@ -37,24 +41,67 @@ def module_org():
...
+        compute_resource = entities.DockerComputeResource(
+            location=[module_loc],
+            organization=[module_org],
+            name=gen_string('alpha'),
+            url='http://{0}:2375'.format(docker_host.ip_addr),
+        ).create()
+        lce = entities.LifecycleEnvironment(organization=module_org).create()
+        repo = entities.Repository(
+            content_type='docker',
+            docker_upstream_name='busybox',
```
Comment (mirzal) [inline tests/foreman/ui_airgun/test_container.py]: You can use `DOCKER_UPSTREAM_NAME` from robottelo.constants.

**Comment by oshtaier** [comment]
okay, so bug is closed and it seems that functionality is going to be removed. So what do we do?

**Comment by abalakh** [comment]
@mirzal when the feature will be dropped, in 6.5 GA or in next release? If in the next one, then there's still value in these tests for 6.5 branch.

**Comment by mirzal** [comment]
I kind of see that value, but in my personal opinion, it's really low.

Let's say that sometime in the future that automation fails. If it is problem with automation, we will have to spend time fixing it, already knowing that these cases are going to be removed in foreseeable future. If this is problem with product, we should report the bug. Which most likely will be CLOSED WONTFIX, just like all the other bzs around that feature created in past 6 months or so.

Of course there is a chance that this automation will find issue in product that PM will decide to fix.
And this code is already written, so if it's stable, there is very little harm in merging it.

@oshtaier @abalakh do we want to let someone from management to make a decision here for us?

**Comment by oshtaier** [comment]
Even if test automation code is stable, corresponding part of application is not maintained and fall apart with each passing day.
I think it will not harm to hear other people opinions

**Comment by abalakh** [comment]
@abalakh like Matt said today on SatQE Team meeting, it's up to component owner to decide which tests are redundant for specific feature and may be dropped. Fortunately you're component owner for container management, so it's up to you to decide whether we should keep them or drop :)
Basically both points sound reasonable to me, i'm totally fine with keeping the tests or updating PR to test removal only.

**Comment by oshtaier** [comment]
I am fine with two options or at least good to keep at least most critical test cases

**Comment by mirzal** [comment]
@abalakh can you leave only these three cases?

* `test_positive_create_with_compresource`, 
* `test_positive_power_on_off`
* `test_positive_delete`

I think they cover the most critical part of feature.

Can you remove others in additional commit in this PR? Then, during merge, we should use "Rebase and Merge" (not Squash). This way we will have commit with other ported cases in git history, so we will be able to retrieve them, if they are ever needed.

**Comment by abalakh** [comment]
@mirzal thanks, addressed.

**Pull request merged**